packages/mux-player/src/index.ts

```diff
@@ -568,6 +562,18 @@ class MuxPlayerElement extends VideoApiElement {
         break;
       }
     }
+
+    const shouldClearState = [
```

Comment:
question(blocker): Why was this change necessary? Feels out of scope. (or is this is rebase diff issue?)

Reply:
I moved this code block from above to below the switch block because once we `render()` we want the `controlsList` to be updated with the new value. it's used in template.ts to render the styles

```js
    ${[...props.controlsList]
      .map((token) => controlsListStyles[token])
      .filter(Boolean)
      .join('\n')}
```